specs/deneb/p2p-interface.md

```diff
@@ -35,17 +37,40 @@ The specification of these changes continues in the same format as the network s
 
 | Name                                     | Value                             | Description                                                         |
 |------------------------------------------|-----------------------------------|---------------------------------------------------------------------|
-| `MAX_REQUEST_BLOBS_SIDECARS`             | `2**7` (= 128)                    | Maximum number of blobs sidecars in a single request                |
-| `MIN_EPOCHS_FOR_BLOBS_SIDECARS_REQUESTS` | `2**12` (= 4096 epochs, ~18 days) | The minimum epoch range over which a node must serve blobs sidecars |
+| `MAX_REQUEST_BLOCKS_DENEB`               | `2**7` (= 128)                    | Maximum number of blocks in a single request                        |
+| `MAX_REQUEST_BLOB_SIDECARS`              | `2**7` (= 128)                    | Maximum number of blob sidecars in a single request                 |
+| `MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS` | `2**12` (= 4096 epochs, ~18 days) | The minimum epoch range over which a node must serve blob sidecars |
 
 ## Containers
 
-### `SignedBeaconBlockAndBlobsSidecar`
+### `BlobSidecar`
 
 ```python
-class SignedBeaconBlockAndBlobsSidecar(Container):
-    beacon_block: SignedBeaconBlock
-    blobs_sidecar: BlobsSidecar
+class BlobSidecar(Container):
+    block_root: Root
```

Comment:
Should we use `BlobIdentifier` here?

Reply:
they're (intentionally) layout-compatible, it doesn't make much practical difference - no strong opinion either way.